src/sdl_utils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "sdl_gl.h"
+#include <iostream>
+
+void sdl_application_abort(const char * msg)
+{
+    std::cout << SDL_GetError() << std::endl;
+    SDL_Quit();
+    exit (-1);
+}
+
+void getInfo(void)

```

Comment:
How about "PrintSDLInfo"  (I don't want to prefix with "SDL" like SDLFunctionName because it's easily confused with the library functions).

And also we're not really "getting" it, as it's not returned.

It's fine as printf for now, with stderr, or cerr <<

But frankly I should add a logging method, so that messages can be printed with a specific level of verbosity (Debug, Info, Warning, Error) etc, which filter based on a flag indicating desired verbosity.